db/models.py

```diff
@@ -49,4 +59,61 @@
     )
 
     def __str__(self) -> str:
-        return f"{self.movie.title} {str(self.show_time)}"
+        return f"{self.movie.title} {self.show_time}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
```

Comment:
The `class Meta` should be defined after all attributes of the `Order` class. Currently, it is correctly placed after the `user` attribute, which is acceptable. However, ensure that all attributes are defined before `class Meta`.